test/TesterInternal/GeoClusterTests/MultiClusterRegistrationTests.cs

```diff
@@ -9,6 +9,7 @@
 using Tests.GeoClusterTests;
 using Xunit;
 using Xunit.Abstractions;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
```

Comment:
I was using MSTest asserts for convenience, because they have an overload that lets me pass a string message. I was not aware that we are trying to move away from them. I suppose I can create a helper method for this functionality.